clang/test/SemaSYCL/scheduler_target_fmax_mhz.cpp

```diff
@@ -10,20 +10,16 @@ template <int N>
 [[intel::scheduler_target_fmax_mhz(N)]] void zoo() {}
 
 int main() {
-  // CHECK-LABEL:  FunctionDecl {{.*}}test_kernel1 'void ()'
```

Comment:
Was this change intentional? `CHECK-LABEL` is used to "anchor" FileCheck's matching behavior, so when you remove the `-LABEL` part, it can start to match in surprising locations you weren't expecting. I don't see any changes that would necessitate dropping the `-LABEL` (here or below), so I'm kinda curious what's going on.

Reply:
I thought it wasn't required, but I can add it back

Reply:
I don't think it's *required* (you can write a test without ever using it, certainly), but I think it makes the test a bit less fragile. I know I've been bitten by tests that accidentally passed because a `CHECK` line was finding something unexpected later in the file and using a `CHECK-LABEL` beforehand would have made the failure really obvious.

I hate FileCheck despite thinking it's pretty powerful. :-D